src/styles/components/committees/committee.module.scss

```diff
@@ -0,0 +1,23 @@
+@import url('https://fonts.googleapis.com/css?family=Be+Vietnam:wght@200..900');
+
+.committee_card {
```

Comment:
might need to make the committee cards a little bigger because despite the figma numbers, the proportions look different on rendering when the screen size is larger than 1500px. once you implement the div around the image thing from sam's example, maybe try setting the card's width to be a percentage of the parent div's width and set an aspect ratio to let the height adjust itself as the width changes for the different screen sizes. try and see if it looks better otherwise this is fine too.